src/lib/virtio/net_driver.lua

```diff
@@ -52,7 +51,6 @@ function VirtioNetDriver:new(args)
 
    if args.use_checksum then
       self.transmit = self._transmit_checksum
-      self.want_features = self.want_features + C.VIRTIO_NET_F_CSUM
```

Comment:
The feature is very confusing -- it allows the guest to offload checksumming to the host (or not), and doesn't necessarily imply that checksums are needed (or not).  In practice we control whether we need to do checksums via the args.use_checksum flag, which is off by default.